tests/acceptance/features/bootstrap/TrashbinContext.php

```diff
@@ -374,11 +375,12 @@ public function theTrashbinDavResponseShouldContainTheseNodes(TableNode $table):
 	 * @param string $user user
 	 * @param string|null $asUser - To send request as another user
 	 * @param string|null $password
+	 * @param bool|null $nonExistentUser (if trying to request on behalf of a non-existent user)
```

Comment:
the default is false so I think, better to not allow null
```suggestion
	 * @param bool $nonExistentUser (if trying to request on behalf of a non-existent user)
```